src/BuildScriptGenerator.Common/BuildScriptGenerator.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Just to be sure I get it right: so basically at this point we're moving to .NET 7, and not .NET 6? Sounds good to me, but I just want to be sure that's the case.

Reply:
yep, there's some dependency compatibility issue with packages. ww've updated system packages to version 7.0.0, which some of them have complaints about using net6.0.

Reply:
Thanks! So to be 100% sure: in practice will we have any projects relying on .NET 6 or will they all move to .NET 7?

Reply:
For the sake of compatability, i would say moving to net7 since most of the packages we're using already released ver 7. But i would like to hear more opinions from team just to make sure we're making the right move.